akka-stream-testkit/src/main/scala/akka/stream/testkit/scaladsl/StreamTestKit.scala

```diff
@@ -43,7 +45,9 @@ object StreamTestKit {
   /** INTERNAL API */
   @InternalApi private[testkit] def assertNoChildren(sys: ActorSystem, supervisor: ActorRef): Unit = {
     val probe = TestProbe()(sys)
-    probe.within(5.seconds) {
+    val c = sys.settings.config.getConfig("akka.stream.testkit")
+    val timeout = c.getDuration("all-stages-stopped-timeout", MILLISECONDS).millis
```

Comment:
don't really like reading config properties from arbitrary places but introducing some StreamTestKitSettings for this seems like overkill